leapp/repository/__init__.py

```diff
@@ -166,7 +167,9 @@ def _load_modules(self, modules, prefix):
         directories = [os.path.join(self._repo_dir, os.path.dirname(module)) for module in modules]
         prefix = prefix + '.' if not prefix.endswith('.') else prefix
         for importer, name, ispkg in pkgutil.iter_modules(directories, prefix=prefix):
-            importer.find_module(name).load_module(name)
+            loader = importer.find_spec(name).loader
+            module = importlib.util.module_from_spec(loader)
+            loader.exec_module(module)
```

Comment:
```suggestion
            spec = importer.find_spec(name)
            module = importlib.util.module_from_spec(spec)
            spec.loader.exec_module(module)
```

Reply:
Thinking about it, I guess you will have to version switch it here, due to py27 backwards compatibility, unless you're no longer supporting py27/el7